### PR TITLE
DPC-1010 Vendor Id Column

### PR DESCRIPTION
**Pull request by Sun-Mountain**
**Why**

Breaking up the vendor workflow ticket into smaller, more manageable sized tickets. This was just adding a column to the `organizations` table.

I also fixed a line the DPC-APP's README.md file.

**What Changed**

Added a column for vendor ids that will be created when an external user creates a vendor account. (The next ticket will be adding the generated id number to relevant accounts.)

Corrected the app's main README file to include the correct `docker-compose up` instructions.

**Choices Made**

`venid` to keep it short but recognizable and different from `npi` and `id` columns.

**Tickets closed**:

DPC 1010

**Future Work**

Will continue with the vendor workflow ticket DPC-997.

**Checklist**

- [x] All tests are passing via `make ci-app` (app change) and `make ci-web` (website change)
- [x] Swagger documentation has been updated
- [x] FHIR documentation has been updated
- [x] Any required dpc-ops changes have a PR submitted and mentioned in this ticket
- [x] Any manual migration steps are documented, scripts written (where applicable), and tested
- [x] Before merging, any required dpc-ops changes have been approved and merged into master of the dpc-ops repo

### CODE REVIEW

File: dpc-web/db/migrate/20200203201716_add_vendor_id_to_organizations.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddVendorIdToOrganizations < ActiveRecord::Migration[5.2]
+  def change
+    add_column :organizations, :venid, :string
```
Comment (switzersc-usds) [inline dpc-web/db/migrate/20200203201716_add_vendor_id_to_organizations.rb]: Let's not shorten names of things unless a) the alternative is quite long and b) the shortened version is clear. In this case, `vendor_id` isn't long and it's much clearer than `venid`.

File: dpc-web/db/migrate/20200204152555_change_vendor_id_column_name.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class ChangeVendorIdColumnName < ActiveRecord::Migration[5.2]
```
Comment (nickrobison-usds) [inline dpc-web/db/migrate/20200204152555_change_vendor_id_column_name.rb]: Do we really need a second migration to rename the column? Since the change hasn't been deployed yet, I think we can just leave this as a single column add.

File: dpc-web-test.sh [hunk 3]
```diff
@@ -12,7 +12,7 @@ make website
 
 # Run the tests
 docker-compose -f dpc-web/docker-compose.yml up start_core_dependencies
-docker-compose -f dpc-web/docker-compose.yml run web bundle exec rails db:create db:migrate db:seed
+docker-compose -f dpc-web/docker-compose.yml run web bundle exec rails db:create db:migrate RAILS_ENV=test
```
Comment (nickrobison-usds) [inline dpc-web-test.sh]: Can you explain why this change is needed now? Has it not been working previously?

Comment (Sun-Mountain) [reply dpc-web-test.sh]: Travis didn't pass the last check.

The error it threw was 

```
** [NewRelic] FATAL : Config file at /dpc-web/config/newrelic.yml doesn't include a 'test' section!
** [NewRelic] FATAL : Config file at /dpc-web/config/newrelic.yml doesn't include a 'test' section!
Migrations are pending. To resolve this issue, run:
        bin/rails db:migrate RAILS_ENV=test
```

@switzersc-usds gave me the code to fix it. As for why, I can't answer that.

Comment (nickrobison-usds) [reply dpc-web-test.sh]: Ok. Sounds good. Thanks!

Comment (switzersc-usds) [reply dpc-web-test.sh]: This wasn't necessary before because the development DB and the test DB were the same. When we updated the db config to use separate DBs for dev vs test, we needed to update travis to make sure the test database gets migrated before running tests.

Comment (switzersc-usds) [reply dpc-web-test.sh]: Also removed `db:seed` because that's not necessary for tests.
